packages/babel-plugin-transform-destructuring/test/fixtures/destructuring/array-unpack-optimisation/output.js

```diff
@@ -21,7 +21,7 @@ var _ref4 = [a[1], a[0]];
 a[0] = _ref4[0];
 a[1] = _ref4[1];
 
-var _ref5 = babelHelpers.toConsumableArray(foo).concat([bar]),
+var _ref5 = Array.prototype.concat(babelHelpers.toConsumableArray(foo), [bar]),
```

Comment:
I'm not sure I love the idea of relying on the implicit `Array.prototype` as the `this` context for the `concat` call.

Reply:
Should i change it to `[].concat(..)` ?

Or i guess we could do 
```
var _a = babelHelpers.toConsumableArray(foo);
Array.prototype.concat.call(_a, _a, ..);
```

Reply:
Okay it is now transforming to `[].concat()` again.